shell/platform/android/platform_view_android.cc

```diff
@@ -414,6 +414,7 @@ void PlatformViewAndroid::InstallFirstFrameCallback() {
 
 void PlatformViewAndroid::FireFirstFrameCallback() {
   jni_facade_->FlutterViewOnFirstFrame();
+  delegate_.GetIsGpuDisabledSyncSwitch()->SetSwitch(true);
```

Comment:
May be splittable to another patch.

Reply:
Need to do something similar on other platforms if we make the change in shell.cc as part of this patch.